File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
// Ask the user the height int between 1 and 8
    int H; 
    do
    {
        H = get_int("Heigh between 1 and 8: ");
    }
    while (H < 1 || H > 8);
// Print space bars and count them in a inverse way than hashs
    for (int i = 1; i <= H; i++)
    {
        for (int j = 0; j < 1 * H - i ; j++)
        {
            printf(" ");
        }
        //Print the hashs
        for (int j = 0; j < 1 * i ; j++)
        {    
            printf("#");
        }
            printf("  ");
        {
              for (int j = 0; j < 1 * i ; j++)
            printf("#");
            printf("\n");
        
        }
    }        
}

File: mario.py
from cs50 import get_int

def main():
    #
    i = get_positive_1_to_8_int()
    for a in range (i):
        for b in range (1*i-a-1):
            print(" ", end ="")
        for b in range (1*a+1):
            print("#", end ="")
        print("  ", end ="")
        for b in range (1*a+1):
            print("#", end ="")
        print()

def get_positive_1_to_8_int():
    while True:
        n = get_int ("Height: ")
        if n > 0 and n < 9:
            break
    return n

main()
